src/views/Pipeline.vue

```diff
@@ -117,6 +117,48 @@ export default defineComponent({
         event.target.complete();
       })
     },
+    async skipJob (job: any) {
+      const alert = await alertController
+        .create({
+          header: this.$t('Skip job'),
+          message: this.$t('Skipping will run this job at the next occurance based on the temporal expression.'),
+          buttons: [
+            {
+              text: this.$t("Don't skip"),
+              role: 'cancel',
+            },
+            {
+              text: this.$t('Skip'),
+              handler: async () => {
+                let skipTime = {};
+                const integer1 = this.temporalExpr(job.tempExprId).integer1;
+                const integer2 = this.temporalExpr(job.tempExprId).integer2
+                if(integer1 === 12) {
+                  skipTime = { minutes: integer2 }
+                }
+                else if (integer1 === 10) {
+                  skipTime = { hours: integer2 }
+                }
+                else if (integer1 === 5) {
+                  skipTime = { days: integer2 }
+                }
+                const time =  DateTime.fromMillis(job.runTime).diff(DateTime.local()).plus(skipTime);  
+                const timeDiff = job.runTime - DateTime.local().toMillis()
```

Comment:
I think timeDiff should not be needed. We will set date time based upon last set runtime. Let me know if I am missed something here

Reply:
Yes Sir, updated the code as per your suggestion.